cmd/cli/namespace_list.go

```diff
@@ -95,16 +95,16 @@ func (l *namespaceListCmd) run() error {
 	return nil
 }
 
-func (l *namespaceListCmd) selectNamespaces() (*v1.NamespaceList, error) {
+func selectNamespacesMonitoredByMesh(meshName string, clientSet kubernetes.Interface) (*v1.NamespaceList, error) {
```

Comment:
Why is `selectNamespacesMonitoredByMesh` no longer a method on `namespaceListCmd`?

Reply:
`selectNamespacesMonitoredByMesh` is now used in `cmd/cli/util.go` as well, so it is not longer a method on `namespaceListCmd`.

https://github.com/johnsonshi/osm/blob/30bf92ae37bc3f103e0d6eb1201904f0e62cf07a/cmd/cli/util.go#L111-L117